src/applications/vaos/routes.jsx

```diff
@@ -4,6 +4,26 @@ import asyncLoader from 'platform/utilities/ui/asyncLoader';
 import AppointmentsPage from './containers/AppointmentsPage';
 import VAOSApp from './containers/VAOSApp';
 import ErrorBoundary from './components/ErrorBoundary';
+import { captureError } from './utils/error';
+import LoadingIndicator from '@department-of-veterans-affairs/formation-react/LoadingIndicator';
+import ErrorMessage from './components/ErrorMessage';
+import FullWidthLayout from './components/FullWidthLayout';
+
+// Handles errors loading components by doing one page reload and showing
+// an error message otherwise
+function handleLoadError(err) {
```

Comment:
I'm not really sure how to test this, unless I export it/move it to a different file.